ivy.xml

```diff
@@ -0,0 +1,6 @@
+<ivy-module version="2.0">
+    <info organisation="com.github.wolfgangmm" module="monex"/>
```

Comment:
I presume the @organisation value should now be `com.github.exist-db` too.

Reply:
Ah right. I have fixed that now.